### ISSUE TITLE
Global Elasticsearch namespaces to different indexes [2.0.6]

### ISSUE THREAD

**Issue by mitchellmaler** [rancher/rancher#14875]
Hello,

I was wondering if it is possible to setup elasticsearch logging to send namespaces to different indexes? My situation is most teams have a -stage and -prod namespace for their application. I am hoping to send all -prod namespace logs to a rolling production index that is stored for a long time but send all -stage to a shot lived index. This way on the elasticsearch side I can clean up the rolling stage indexes but keep the prod indexes until their age out time.

**Comment by massimo-cannavo** [comment]
There is support for **project logging**. Meaning you could create a project for production and staging. Then you can have different indexes for each project. On the namespace level, I believe that is not currently supported yet. **Is that a feature that you would be interested in?** There is also cluster level logging, which would pollute your logs with everything from the cluster. But you can also use another index for the cluster level, preventing the pollution of your logs.

**TL;DR** 

Project logging might solve your issue. You would have to **move/migrate** workloads to different projects.

**Comment by mitchellmaler** [comment]
We do have (plan) on teams having their own projects and their namespaces under those would have thier prod and non-prod namespaces for their different applications. I was thinking setting the logging at the global level that way teams logs would automatically end up in our single large elasticseach cluster. Even if we did it per project it would include both their prod and non prod namespace and our rentention for non-prod is short compared to prod logs. Elasticseach recommends using the same index as much as possible but just rolling it over and use different queries to find your data so we only have a cluster, non-prod, and prod index that are all rolling.

Using fluent pipelines might work where I can setup logic based on the nanespace name (*-stage) goes to non prod index, etc. When is this going to be included?

**Comment by loganhz** [comment]
Fluentd aggregator is available in v2.1.1
